lib/Interpreter/IncrementalParser.h

```diff
@@ -175,6 +175,14 @@ namespace cling {
       return m_Transactions.back();
     }
 
+    ///\brief Merge transactions after the one given.
+    ///
+    ///\param[in] T - transactions after which to merge
+    ///\param[in] prev - merge into transaction prior to T or T itself
+    ///\returns the parent transaction of the merge.
+    ///
+    const Transaction* mergeTransactionsAfter(const Transaction *T, bool prev);
```

Comment:
Okay, we don't agree. Let's try from a different angle: do you actually need that argument? All calls seem to pass `true`. Can we just remove the argument and rename this to `mergeTransactionsInfoPrevious(const Transaction *T)`?